synapseclient/models/protocols/form_protocol.py

```diff
@@ -6,15 +6,14 @@
         FormGroup,
         FormData,
     )
-    from synapseclient.models.mixins.form import StateEnum
 
 from synapseclient.core.async_utils import wrap_async_generator_to_sync_generator
 
 
 class FormGroupProtocol(Protocol):
     """Protocol for FormGroup operations."""
 
-    def create(
+    def create_or_get(
```

Comment:
@BryanFauble/ @linglp  does the rest of the client follow `create_or_get`? or is the expected that when you try to create a form that already exists that it will return the form.